### PR TITLE
ES-1837: Corda Runtime Gradle Plugin uses Combined Worker Docker image

### PR DESCRIPTION
**Pull request by tlawson3**
https://r3-cev.atlassian.net/browse/ES-1837

Using the docker image to run Corda rather than the Combined Worker JAR file.
Deleted all the code relating to downloading the JAR, plus we no longer need to get the JDBC driver after https://github.com/corda/corda-runtime-os/pull/5587

Updated the tests to use a 5.2 beta version, as the 5.0.1.0 docker image doesn't have the drivers JAR, and I don't think we should have tests which mount the driver when the main code doesn't need it.

Tested locally
Screenshot 2024-02-05 at 15 40 10

### CODE REVIEW

File: tools/corda-runtime-gradle-plugin/README.md [hunk 1]
```diff
@@ -7,20 +7,85 @@ Add the following extension properties
...
+               "-mbus.busType=DATABASE",
+               "-spassphrase=password",
+               "-ssalt=salt",
+               "-ddatabase.user=user",
+               "-ddatabase.pass=password",
+               "-ddatabase.jdbc.url=jdbc:postgresql://postgresql:5432/cordacluster",
+               "-ddatabase.jdbc.directory=/opt/jdbc-driver/"
+    ]
+    ports:
+      - 8888:8888
```
Comment (anton-subbotin) [inline tools/corda-runtime-gradle-plugin/README.md:62]: Among these settings, REST ports configuration should match what is given in `cordaClusterURL` plugin config parameter; maybe worth adding it in the example (lines 9 to 24), explicitly specifying smth like `https://localhost:8888` ?

**Review by davidcurrie — DISMISSED** [review]
LGTM

File: tools/corda-runtime-gradle-plugin/README.md (hunk 1 above)
Comment (tlawson3) [reply tools/corda-runtime-gradle-plugin/README.md:62]: I'm not sure I understand your suggestion.
I understand the ports should match the `cordaClusterURL`, but what change are you wanting me to make?
You want me to add the `cordaClusterURL` setting to the cordaRuntimeGradlePlugin config block in the README?
Or would you like me to add a comment next to the compose file saying "Ensure the port given in the compose file matches the cordaClusterURL setting in the cordaRuntimeGradlePlugin extension properties; default is 8888"

Comment (anton-subbotin) [reply tools/corda-runtime-gradle-plugin/README.md:62]: A comment would be enough 👍

**Review by anton-subbotin — DISMISSED** [review]
LGTM

File: tools/corda-runtime-gradle-plugin/README.md [hunk 2]
```diff
@@ -14,6 +14,9 @@ Add the following extension properties
         vnodeRegistrationTimeout = "60000"
         cordaProcessorTimeout = "300000"
         workflowsModuleName = "workflows"
+        cordaClusterURL = "https://localhost:8888"
+        cordaRpcUser = "admin"
+        cordaRpcPasswd ="admin"
```
Comment (davidcurrie) [inline tools/corda-runtime-gradle-plugin/README.md]: I hadn't spotted this before, but can we call these something like `cordaRestUser` or `cordaRestApiUser`?

**Review by anton-subbotin — APPROVED** [review]

**Review by davidcurrie — APPROVED** [review]
LGTM

**Pull request merged**